alma/controllers/hook/DisplayPaymentHookController.php

```diff
@@ -86,9 +86,6 @@ public function run($params)
             $isInstallmentAccordingToDeferred = $isDeferred ? $installment === 1 : $installment !== 1;
 
             if ($isInstallmentAccordingToDeferred) {
-                if (isset($paymentOptions[$key]) && isset($paymentOptions[$key]['disabled']) && $paymentOptions[$key]['disabled'] === true) {
```

Comment:
won't removing this put back another bug you fixed a while ago ?

Reply:
@PaulRouss3l 
I think the bug fix should not be here but in the other file I fixed. The bug was still there in some cases and now it is not. I made a comparison with the data I received before the refactoring.

Reply:
Hi benjamin, to ensure this fix will not broke another thing, could you develop test(s) (unit and/or E2E) to validate the 2 uses cases are now working fine.

There is at least to tests to write:

1. one for the current bug
2. one for the previous you tried to fix as Paul said

So:
* without this fix, one the first test should be fail
* without previous fix, the first test should green and the second one should be produce a red exit
* with the 2 fix applied, the 2 tests should be green

Reply:
Hello @syjust-alma 
The fix was for the same issue.
We did the first correction thinking that fix the issue, but in fact we wrong identify the issue and the second correction is the right fix.
But I can try to add test for confirm the fix yes.